types.go

```diff
@@ -6,6 +6,49 @@ import (
 	"math/bits"
 )
 
+// Constants for looking up Pokemon stats
+const (
+	STAT_HP = iota
+	STAT_ATK
+	STAT_DEF
+	STAT_SPATK
+	STAT_SPDEF
+	STAT_SPD
+)
+
+// Constants for status effects on a Pokemon
+const (
+	STATUS_BURN = 1 << iota
+	STATUS_FREEZE
+	STATUS_PARALYZE
+	STATUS_POISON
+	STATUS_BADLY_POISON
+	STATUS_SLEEP
+)
+
+// Constants for IVs and EVs
+const (
+	MAX_FRIENDSHIP    = 255
+	MAX_EV            = 255
+	MAX_IV            = 31
+	MAX_STAT_MODIFIER = 6
+	MIN_STAT_MODIFIER = -6
+	TOTAL_EV          = 510
+)
```

Comment:
I feel like this should stay in `pokemon.go`, since these constants relate directly to the fields of that type. Maybe we can leave out `MAX_STAT_MODIFIER` and `MIN_STAT_MODIFIER`, but I think the rest should stay in `pokemon.go`